### ISSUE TITLE
Cifar10 experiments

### ISSUE THREAD

**Issue by kilianFatras** [sbnietert/robust-OT#1]
Hi,

Thank you very much for realizing the code. I would like to know if you intend to release the Cifar10 code of your paper.

Best,

Kilian

**Comment by sbnietert** [comment]
Hi, I am currently traveling but can attend to this by Monday. This code was intended to be included, sorry for any inconvenience.

**Comment by kilianFatras** [comment]
Hi,

No problem. Thank you very much for releasing the code.

Best,

Kilian

**Comment by sbnietert** [comment]
I found the relevant code - will add it this to the repo by tomorrow, though I want to type up a few notes so that it is easily reproducible before pushing. But the details are simple enough if you would like to go ahead yourself - as mentioned in B.4 of our appendix, we used the control GAN of Balaji et al. (https://github.com/yogeshbalaji/robustOT) without robustification weights enabled and 5% contamination. (At a glance, I think running the command `python main.py --base_cfg_path configs/experiments/CIFAR10_MNIST.json --cfg_path configs/unconditional/WGAN.json` from the GAN folder should do the trick, but I can provide full instructions tomorrow). For the robust GAN, we made the following adjustment 
![image](https://user-images.githubusercontent.com/9258182/182259164-311a069c-05a8-4f41-ac84-9d9ac55e3541.png)
to their trainer.py (https://github.com/yogeshbalaji/robustOT/blob/main/GAN/trainer.py).
